_sass/layout/home.scss

```diff
@@ -11,8 +11,8 @@
   .page-item {
     .page-link {
       color: var(--btn-patinator-text-color);
-      width: 2.5rem;
-      height: 2.5rem;
+      width: 2rem;
+      height: 2rem;
```

Comment:
Simply reducing the size of the button is not a good option as it is more likely to be touched by mistake on a small width screen. It might be better to reduce the number of the pagination buttons, or use `<select>` tag. I will improve the design when I have time.

Anyway, thank you for your PR!